nexus/src/app/background/tasks/region_replacement.rs

```diff
@@ -74,16 +74,13 @@ impl BackgroundTask for RegionReplacementDetector {
                 Ok(regions) => regions,
 
                 Err(e) => {
-                    error!(
-                        &log,
+                    let s = format!(
                         "find_regions_on_expunged_physical_disks failed: {e}"
                     );
-                    err += 1;
+                    error!(&log, "{s}");
+                    status.errors.push(s);
```

Comment:
take it or leave it: it might be worth making the error be a structured field on the log entry, so that it can be easily searched with `looker`. maybe consider something like:
```rust
let message = "find_regions_on_expunged_physical_disks failed";
error!(&log, "{message}"; "error" => e);
status.errors.push(format!("{message}: {e}");
```

but, it's not a huge deal.

Reply:
in this case, I'm also going to leave it the way it is - I noodled a bit with this suggestion but didn't like the repetition.

Reply:
cool, fine with me!